js/app.jsx

```diff
@@ -92,6 +92,15 @@ function cleanJobPath(path) {
   return path;
 }
 
+function unique(array) {
```

Comment:
this is O(N^2), but I guess it is okay.  An O(N) approach is to build a map of item to true or something (simulate a set) then use that to check for existence.